Makefile

```diff
@@ -133,6 +136,17 @@ deployment/%/overlay.jsonnet:
 show-secret:  ## Show currently deployed foxtrot auth secret
 	kubectl get secret -n foxtrot foxtrot -o go-template='{{.data.authsecret | base64decode}}{{"\n"}}'
 
+PAYLOAD = \
+{ \
+	"command": "kubecfg update $(TLA_ARGS) https://github.com/foxygoat/foxtrot/raw/$(REF)/deployment/main.jsonnet", \
```

Comment:
then this could also use `$(GITHUB_REPO)/raw/$(REF)/$(MAIN)` or something

Reply:
yeah agreed.

different PR.